docs/sdks/best-practices/non-blocking-initialization.md

```diff
@@ -9,22 +9,11 @@ title: Initialize before assigning
 Don't be blocked! We're here to help you get up and running with Eppo. Contact us at [support@geteppo.com](mailto:support@geteppo.com).
 :::
 
-Eppo's SDK initialization is non-blocking by default. This means your application can continue running while the SDK initializes in the background. However, if you call `get<Type>Assignment` before initialization completes, you may get unexpected results due to race conditions.
-
-### What Happens If You Don't Wait
-When you call `get<Type>Assignment` before initialization completes, one of two things will happen:
-
-1. If the SDK has a cached configuration in persistent storage (like localStorage in JavaScript):
-   - It will use the previous configuration
-   - This might not include your latest changes
-
-2. If there's no cached configuration:
-   - The SDK will return your specified default value
-   - Your feature flags won't work as expected
+Eppo's SDK initialization is non-blocking by default. This means your application can continue running while the SDK initializes in the background. However, if you call `get<Type>Assignment` before initialization completes, the functions will return your specified default values.
```

Comment:
Fixed this part as it was factually incorrect: SDKs that have persistent storage race initialization from persistent storage and remote config, so you have to wait for initialization to complete in either case. You always get default variation if SDK has not finished initialization.